recipes/yaehmop/LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+Copyright (C) 1995 Greg Landrum
```

Comment:
The license is in downloaded source already, so you do not need to include it here.

Reply:
Thus you should delete this file.